tests/Makefile

```diff
@@ -0,0 +1,14 @@
+SCRIPTS = \
+  build-viewer-reports \
+  compare-result \
+  compare-source \
+  compare-viewer-reports
+
+default:
+	@ echo Nothing to do for $@
```

Comment:
We should add a comment explaining why `default` is set to nothing. Something like "makes sure users are ware which operation is performed by this Makefile".